### PR TITLE
Blobs recheck their collision with player after they crouched

### PR DESCRIPTION
**Pull request by CatWiz**
## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

Crouching makes blobs that are touching player blob recheck their collisions with it. Fixes problem when crouching doesnt do anything if enemy keeps walking into you after touching you/is standing on you.

### CODE REVIEW

**Comment by eps0003** [comment]
You should also do the same when a player dies. I've had situations where I cannot walk past a dead body because I killed them while colliding with them.

**Comment by CatWiz** [comment]
RunnerDeath already does checkCollisionsAgain for the body. Should it make all blobs that are touching it recheck their collisions instead?

**Comment by eps0003** [comment]
Not sure what needs to be done to fix it. I just wanted to make it known that similar behaviour occurs with dead bodies.

File: Entities/Characters/Scripts/CrouchCommon.as [hunk 1]
```diff
@@ -12,3 +12,15 @@ bool isCrouching(CBlob@ this)
...
+bool hasJustCrouched(CBlob@ this)
+{
+	return
+	this.isOnGround()
+	&& this.isKeyJustPressed(key_down)
+	//cannot have movement intent
+	&& !this.isKeyPressed(key_left)
+	&& !this.isKeyPressed(key_right)
+	//cannot have banned crouch (done in actor logic scripts)
+	&& !this.hasTag("prevent crouch");
```
Comment (eps0003) [inline Entities/Characters/Scripts/CrouchCommon.as:25]: There's no need to repeat the same chunk of code
```suggestion
	return isCrouching(this) && this.isKeyJustPressed(key_down);
```

**Review by eps0003 — CHANGES_REQUESTED** [review]

**Pull request merged**